python/cuml/test/test_fil.py

```diff
@@ -221,6 +221,21 @@ def test_output_algos(algo, small_classifier_and_preds):
     assert np.allclose(fil_preds, xgb_preds_int, 1e-3)
 
 
+@pytest.mark.skipif(has_xgboost() is False, reason="need to install xgboost")
+@pytest.mark.parametrize('sparse', ['AUTO', 'DENSE', 'SPARSE'])
+def test_output_sparse(sparse, small_classifier_and_preds):
+    model_path, X, xgb_preds = small_classifier_and_preds
+    fm = ForestInference.load(model_path,
+                              algo='NAIVE',
+                              output_class=True,
+                              sparse=sparse,
+                              threshold=0.50)
+
+    xgb_preds_int = np.around(xgb_preds)
```

Comment:
Can you check explicitly that the loaded model is actually sparse?

Reply:
That would require adding more methods to the C++ and Python API. Do you want me to do this (maybe in a separate PR)?

Reply:
I think a separate PR is fine... I do think that exposing that info to higher levels (is_sparse or whatever) will be generally a good idea.